src/index.html

```diff
@@ -14,6 +14,87 @@
   </head>
   <body>
     <h1 class="logo">MA</h1>
-    <script src="scripts/main.js"></script>
+    <div id="notifications"></div>
+    <script>
+      function createNotification(message, type) {
+        const notification = document.createElement('div');
+        notification.className = type;
+        notification.dataset.qa = 'notification';
+        notification.textContent = message;
+        document.getElementById('notifications').appendChild(notification);
+      }
+
+      let firstPromiseResolved = false;
+      let secondPromiseResolved = false;
+      let thirdPromiseResolved = false;
+      let leftClicked = false;
+      let rightClicked = false;
+
+      const firstPromise = new Promise((resolve, reject) => {
+        setTimeout(() => {
+          if (!firstPromiseResolved) {
+            reject('First promise was rejected');
+          }
+        }, 3000);
+      });
+
+      const secondPromise = new Promise((resolve) => {});
```

Comment:
The `secondPromise` is created but never resolved or rejected. This will lead to the promise being pending indefinitely, which can lead to memory leaks or unexpected behavior in your application.